Cargo.toml

```diff
@@ -48,6 +48,8 @@ tracing-subscriber = { version = "0.3", default-features = false }
 utoipa = { version = "4.2.3", default-features = false }
 utoipa-swagger-ui = { version = "7.1.0", default-features = false }
 uuid = { version = "1.10.0", default-features = false }
+deltalake = {version="0.22.0",features=["datafusion"]}
```

Comment:
Probably should use `default-features = false` here for consistency with other dependencies.

Reply:
yes. done with this